src/templates/next/layouts/render.tsx

```diff
@@ -43,6 +44,8 @@ export const renderComponent = ({
       return <CollectionCard {...component} />
     case "contentpageheader":
       return <ContentPageHeader {...component} />
+    case "filter":
+      return <Filter {...component} />
```

Comment:
similar comment left on the CollectionSearch PR if we really consider `Filter` a "component"

Reply:
Let me reply there

Reply:
Converted in 87c41a1a8d9db43baa97121307638aca908fde7c.